src/components/NavBar.jsx

```diff
@@ -0,0 +1,73 @@
+import { Button } from '@headlessui/react';
+import { useNavigate } from 'react-router-dom';
+import { useAuth } from '../context/AuthProvider';
+import cartImage from '../assets/images/cart.png';
+import logout from '../assets/images/logout.png';
+import { useLocation } from 'react-router-dom';
+
+const NavBar = () => {
+  const { isLoggedIn, clearUserSession } = useAuth();
+  const navigate = useNavigate();
+  const location = useLocation();
+
+  const handleNavigation = (link) => {
+    navigate(link);
+  };
+
+  const handleLogout = () => {
+    clearUserSession();
+    navigate('/');
+  };
+
+  return (
+    <>
+      {isLoggedIn ? (
+        <div className="flex items-center gap-2">
+          {location.pathname !== '/cart' && (
+            <Button as="button" onClick={() => handleNavigation('/cart')}>
+              <img className="rounded border border-gray p-1" src={cartImage} />
+            </Button>
+          )}
+          {location.pathname !== '/account' && (
+            <Button
+              as="button"
+              onClick={() => handleNavigation('/account')}
+              className="h-[35px] rounded border border-gray px-1 text-gray"
+            >
+              My Account
+            </Button>
+          )}
+          <Button as="button" onClick={handleLogout}>
+            <img
+              className="h-[35px] rounded border border-gray p-1"
+              src={logout}
+            />
+          </Button>
+        </div>
+      ) : (
+        <>
+          {location.pathname !== '/sign_in' && (
+            <Button
+              as="button"
+              onClick={() => handleNavigation('/sign_in')}
+              className="mx-1 text-gray"
+            >
+              Log In
+            </Button>
+          )}
+          {location.pathname !== '/sign_up' && (
+            <Button
+              as="button"
+              onClick={() => handleNavigation('/sign_up')}
+              className="mx-1 rounded border border-gray px-1 text-gray"
```

Comment:
I might be wrong since I don’t have access to the `dev` mode in Figma, but in the design, the text looks like darkGreen. What do you think about that?

Reply:
Color is 8799A9. All color with description you can see in tailwind.config.js